src/app/features/dialogs/increase-fee-dialog/components/increase-fee-actions.tsx

```diff
@@ -5,11 +5,12 @@ import { useWalletType } from '@app/common/use-wallet-type';
 import { Button } from '@app/ui/components/button/button';
 
 interface IncreaseFeeActionsProps {
-  isDisabled: boolean;
+  isDisabled?: boolean;
+  isBroadcasting?: boolean;
   onCancel(): void;
 }
 export function IncreaseFeeActions(props: IncreaseFeeActionsProps) {
-  const { onCancel, isDisabled } = props;
+  const { onCancel, isDisabled, isBroadcasting } = props;
```

Comment:
to avoid further refactoring, just have put it here. it would be removed in Approval UX